### PR TITLE
Cult Fixes for Construct Icons and Spells

### LINKED ISSUE #27391: Xeno tail sweep button does not turn red when in cooldown.
What it says on the tin.

### PR DESCRIPTION
**Pull request by Robustin**
:cl: Robustin
add: Added a sexy new icon for the harvester's AOE conversion spell
fix: Fixed construct's forcewall being invisible
fix: Fixed cult constructs "Locate Master" and "Locate Prey" not functioning
fix: Fixed spell action buttons not showing their actual availability status
tweak: Changed the duration of a few frames for the new Cult ending
/:cl:

It should also fix #27391

### CODE REVIEW

File: code/game/gamemodes/wizard/soulstone.dm [hunk 1]
```diff
@@ -3,6 +3,7 @@
 	icon = 'icons/obj/wizard.dmi'
 	icon_state = "soulstone"
 	item_state = "electronic"
+	layer = ABOVE_MOB_LAYER
```
Comment (ChangelingRain) [inline code/game/gamemodes/wizard/soulstone.dm]: really weird really weird wee woo wee woo make this HIGH_OBJ_LAYER or something

**Comment by ChangelingRain** [comment]
Fix forcewall being invisible by giving the forcewall `icon = 'icons/effects/cult_effects.dmi'` instead of adding it back to effects.dmi

File: code/game/gamemodes/wizard/soulstone.dm (hunk 1 above)
Comment (Robustin) [inline code/game/gamemodes/wizard/soulstone.dm]: Its fookin' magic I can't explain shit.

**Review by ChangelingRain — CHANGES_REQUESTED** [review]
Fix forcewall being invisible by giving the forcewall `icon = 'icons/effects/cult_effects.dmi'` instead of adding it back to effects.dmi

**Review by ChangelingRain — APPROVED** [review]

**Pull request merged**